packages/ui/Scrollable/Scrollable.js

```diff
@@ -385,13 +399,17 @@ class ScrollableBase extends Component {
 			{hasDataSizeChanged} = this.childRef;
 
 		// Need to sync calculated client size if it is different from the real size
-		if (this.childRef.syncClientSize) {
+		const isSync = this.childRef.syncClientSize ? this.childRef.syncClientSize() : null;
```

Comment:
If `syncClientSize()` MUST be called before handling the initial position prop, let's leave some comment about it for the future.